types/plugin/localeData.d.ts

```diff
@@ -9,12 +9,34 @@ declare module 'dayjs' {
 
   interface InstanceLocaleDataReturn {
     firstDayOfWeek(): number;
-    weekdays(instance?: Dayjs): WeekdayNames;
-    weekdaysShort(instance?: Dayjs): WeekdayNames;
-    weekdaysMin(instance?: Dayjs): WeekdayNames;
-    months(instance?: Dayjs): MonthNames;
-    monthsShort(instance?: Dayjs): MonthNames;
+
+    weekdays(instance: Dayjs): string;
+    weekdays(instance: undefined): WeekdayNames;
```

Comment:
could we remove those lines of `weekdays(instance: undefined)` ? it loos the same with `weekdays()`

Reply:
Thank you for your review.
I fixed it.